ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/websocket/RawWebSocket.kt

```diff
@@ -47,7 +47,7 @@ class RawWebSocket(
     }
 
     override suspend fun close(cause: Throwable?) {
-        terminate()
+        cause?.let { socketJob.completeExceptionally(it) } ?: socketJob.complete(Unit)
```

Comment:
`socketJob` successful completion doesn't close anything, isn't it? Doesn't closing without exception lead to forever hanging raw read/write coroutines?